ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -64,13 +79,46 @@ const styles = StyleSheet.create({
 const LABEL_COL_SIZE_NARROW_LABEL = 1;
 const VALUE_COL_SIZE_NARROW_LABEL = 2;
 
-const LABEL_COL_SIZE_WIDE_LABEL = 1;
+const LABEL_COL_SIZE_WIDE_LABEL = 2;
```

Comment:
What's the reason for this change? The proportions 1/2 : 1/2 should work just fine

Reply:
it give just one problem with the "why?" link. without my update, the link is not on the same row of  the label while the prototype requires them on the same line.

Reply:
does it break some other screen? :)